tests/README.md

```diff
@@ -8,22 +8,43 @@ use mordred to test all data sources supported in GrimoireLab.
 * requirements.cfg: sample file with the versions of GrimoireLab stack to be used
 * orgs_file: organizations file to be loaded in Sorting Hat for affiliations
 * cache-test.tgz: perceval cache for all data sources, mbox data and irc data.
-* docker-compose.yml: sample docker compose file to launch mordred
+* docker-compose.yml: minimum docker compose file to launch mordred
+* docker-compose-local.yml: example of additional local docker compose config file
 * stage: Script for testing for the mordred docker image
 
 So you just need to execute:
 
-```
-mkdir logs
-docker-compose up mordred
+```bash
+$ mkdir logs
+$ docker-compose up mordred
 ```
 
 In the logs/all.log file you can track the execution of mordred.
 
+The above command run the set of containers specified in the default
+configuration file `docker-compose.yml`.
+The one provided in this directory intends to be minimal,
+so that containers can be deployed with as a few constraints as possible.
+For example, it does not expose ElasticSearch or Kibana ports to the host.
+In case you want to do this, for example for accessing them from the host,
+run a local configuration override file. For example:
+
+```bash
+$ docker-compose -f docker-compose.yml -f docker-compose-local.yml up mordred
+```
```

Comment:
Nice, we are just redefining `docker-compose-local.yml`some of the config in `docker-compose.yml` not creating a totally new config file. I find this approach maintainable, so great!